client/src/pages/lab_management/edit_module/EditModulePage.tsx

```diff
@@ -130,7 +130,7 @@ export default function EditModulePage({
       <title>Edit Training | Make @ RIT</title>
       <Typography variant="h4" textAlign="center">Edit {module.name}</Typography>
       <Stack
-        direction="row"
+        direction={isMobile ? "column" : "row"}
```

Comment:
this page crashes when i try to load it complaining that isMobile is never defined. I think you need to import useIsMobile and create the isMobile constant in this file as well